src/main/kotlin/racingcar/view/ResultView.kt

```diff
@@ -0,0 +1,30 @@
+package racingcar.view
+
+import racingcar.domain.Car
+
+class ResultView {
+    fun printRacingResult() {
+        println("실행 결과")
+    }
+
+    fun printCurrentRace(cars: List<Car>) {
+        for (car in cars) {
+            println(mapForDisplay(car.distance))
+        }
+        println()
+    }
+
+    private fun mapForDisplay(distance: Int): String {
+        val stringBuilder = StringBuilder()
+
+        for (i in 1..distance) {
+            stringBuilder.append(DISTANCE_VIEW)
+        }
```

Comment:
repeat()를 사용해보는 것은 어떨까요?
```suggestion
        repeat(distance) { stringBuilder.append(DISTANCE_VIEW) }
```

Reply:
와우...! 👍 최곱니다